web/gui/dashboard_info.js

```diff
@@ -135,7 +135,12 @@ netdataDashboard.menu = {
 
     'mdstat': {
         title: 'MD arrays',
-        icon: '<i class="fas fa-hdd"></i>'
+        icon: '<i class="fas fa-hdd"></i>',
+        info: '<p>RAID devices are virtual devices created from two or more real block devices. '+
+        '<a href="https://man7.org/linux/man-pages/man4/md.4.html" target="blank">Linux Software RAID</a> devices are '+
```

Comment:
Please, add `_` for `target`. I am not sure if `blank` will open pages in another tab like `_blank`.

Reply:
Take a look in all links added with this PR.